01-JS-DOM-APIs/javascript.js

```diff
@@ -0,0 +1,117 @@
+//1-4
+function show(){
+                  let obj = document.getElementById("hidden");
+                  if(obj.style.display == "block"){
+                        obj.style.display = "none";
```

Comment:
instead of manipulating style.display, try to add and remove a *hidden* class to your variable, that way you can add all your necessary styles to that class, think about, what if instead of just changing your display attribute, you should change your font, and your color or add borders as well?